code/modules/medical/surgery_procs.dm

```diff
@@ -329,8 +329,11 @@ var/global/list/chestitem_whitelist = list(/obj/item/gnomechompski, /obj/item/gn
 	DEBUG_MESSAGE("<b>[patient]'s surgery (performed by [surgeon]) damage_low is [damage_low], damage_high is [damage_high]</b>")
 
 /* ---------- SCALPEL - HEAD ---------- */
-
 	if (surgeon.zone_sel.selecting == "head")
+		if (patient.hasStatus("sterilized")) // reduce the rand value by 5 up and down, effectivily reducing it by 10
+			damage_low = calc_surgery_damage(surgeon, screw_up_prob, rand(0,10) * surgCheck/*, src.adj1, src.adj2*/)
+			damage_high = calc_surgery_damage(surgeon, screw_up_prob, rand(10,20) * surgCheck/*, src.adj1, src.adj2*/)
+
```

Comment:
This seems like a very messy way of going about this. Please consider a different approach :/